api/PasswordCredential.json

```diff
@@ -1,156 +1,158 @@
 {
   "api": {
-    "__compat": {
-      "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential",
-      "support": {
-        "chrome": {
-          "version_added": "51"
-        },
-        "webview_android": {
-          "version_added": "51"
-        },
-        "chrome_android": {
-          "version_added": "51"
-        }
-      },
-      "status": {
-        "experimental": true,
-        "standard_track": true,
-        "deprecated": false
-      }
-    },
-    "additionalData": {
+    "PasswordCredential": {
       "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/additionalData",
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential",
         "support": {
           "chrome": {
-            "version_added": "51",
-            "version_removed": "60"
+            "version_added": "51"
           },
           "webview_android": {
-            "version_added": "51",
-            "version_removed": "60"
+            "version_added": "51"
           },
           "chrome_android": {
-            "version_added": "51",
-            "version_removed": "60"
+            "version_added": "51"
           }
         },
         "status": {
           "experimental": true,
           "standard_track": true,
-          "deprecated": true
+          "deprecated": false
         }
-      }
-    },
-    "iconURL": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/iconURL",
-        "support": {
-          "chrome": {
-            "version_added": "52"
-          },
-          "webview_android": {
-            "version_added": "52"
+      },
+      "additionalData": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/additionalData",
+          "support": {
+            "chrome": {
+              "version_added": "51",
+              "version_removed": "60"
+            },
+            "webview_android": {
+              "version_added": "51",
+              "version_removed": "60"
+            },
+            "chrome_android": {
+              "version_added": "51",
+              "version_removed": "60"
```

Comment:
Actually, this was removed but in 64. 
Commit: https://chromium.googlesource.com/chromium/src/+/57b2dc23b962feeadd2dcf60a399c9f38d2087e3%5E%21/#F9
Where it landed: https://storage.googleapis.com/chromium-find-releases-static/57b.html#57b2dc23b962feeadd2dcf60a399c9f38d2087e3